test/kickstart-templates/kickstart.ks.template

```diff
@@ -35,8 +35,30 @@ rootpw --lock
 # Configure ostree
 ostreesetup --nogpg --osname=rhel --remote=edge --url=REPLACE_OSTREE_SERVER_URL --ref=REPLACE_BOOT_COMMIT_REF
 
+# Configure a non-ostree system
+#liveimg --url file:///run/install/repo/liveimg.tar.gz
```

Comment:
Where does it come from?

Reply:
Ah, it's part of the ISO itself, right?
`sed` only uncomments this line, it doesn't change the value of `--url`

Reply:
Yes, exactly. The value is "hardcoded".
If that changes in the future, we can extract this value from the ISO and fill in kickstart dynamically.